src/test/java/com/hivemq/codec/decoder/MQTTMessageDecoderTest.java

```diff
@@ -174,4 +176,149 @@ public void test_second_connect_received() {
         assertFalse(channel.isOpen());
 
     }
+
+    @Test
+    public void test_connect_mqtt5_packet_size_too_large() {
+        final byte[] mqtt5_connect = {
```

Comment:
please make variable-names in camelcase (182,208,234)

Reply:
changed